tsconfig.json

```diff
@@ -11,6 +11,9 @@
         "paths": {
             "@azure/functions": [
                 "types"
+            ],
+            "@azure/functions-worker": [

```

Comment:
We _have_ to provide a worker package that exposes any functionality the worker provides to the user like `registerHook`. The only question is what to name it and how "fancy" it should be. My proposal, as discussed elsewhere, is for all the fancy stuff to be in the separate programming model package. However, the worker package still needs to be exposed and the programming model package would directly reference it. End users _could_ reference the worker package, but the targeted audience of the worker package is just programming model package authors.

Reply:
Yeah that makes sense. So is the plan that later in the future, we would also expose these hooks in the "fancier" programming model package, perhaps adding some fancier methods and syntactic sugar on top of the ones here, and have that package use this API here? Because to me it feels like these pre and post invocation hooks _are_ targeted towards more than just the programming model package authors.

Reply:
Yes that is correct. Importantly though, nothing would prevent us from adding the "fancier methods" to this worker package if we end up deciding we don't want a separate programming model package.